core/src/elements/ons-tabbar/index.js

```diff
@@ -208,7 +208,17 @@ class TabbarElement extends BaseElement {
       this._compile();
     }
 
-    this._contentElement = util.findChild(this, '.tab-bar__content');
+    for (var i = 0; i < this.firstChild.children.length; i++) {
+      this.firstChild.children[i].style.display = 'none';
+    }
+
+    var activeIndex = this.getAttribute('activeIndex');
+
+    if (activeIndex && this.children[1].children.length > activeIndex) {
+      this.children[1].children[activeIndex].setAttribute('active', 'true');
+    }
```

Comment:
@argelius i mean this code, maybe we can do it differently, because this is kind of hacky and won't work if you load pages.